components/UI/image/ImageWithFallback.tsx

```diff
@@ -0,0 +1,38 @@
+import Image from "next/image"
+import { useState, useEffect } from "react"
+
+import FallbackImage from "../../../public/images/common/fallback-image.svg"
+
+type Props = {
+  alt: string
+  src: string
+  width?: number
+  height?: number
+  style?: any
+}
+export const ImageWithFallback = ({
+  alt,
+  src,
+  width,
+  height,
+  style,
+  ...props
+}: Props) => {
+  const [error, setError] = useState(false)
+
+  useEffect(() => {
```

Comment:
what for?

Reply:
Check if theres some kind of error. If so, show the fallback right away